ArcDPS Boon Table/main.cpp

```diff
@@ -229,43 +246,87 @@ uintptr_t mod_combat(cbtevent* ev, ag* src, ag* dst, char* skillname, uint64_t i
 	/* combat event. skillname may be null. non-null skillname will remain static until module is unloaded. refer to evtc notes for complete detail */
 	else
 	{
+		/* default names */
+		if (!src->name || !strlen(src->name)) src->name = (char*)"(area)";
+		if (!dst->name || !strlen(dst->name)) dst->name = (char*)"(area)";
```

Comment:
Why changing the name? when the name is null, you can just skip the npc detection. Seems like this is again something you forgot to remove from the example^^

Reply:
Yeah, this was part of the example code, I'll remove it

Reply:
Done.